src/session.rs

```diff
@@ -3235,6 +3256,15 @@ impl Session {
             Command::SelectionYank => {
                 self.yank_selection();
             }
+            Command::SelectionFlip(dir) => {
+                self.flip_selection(dir);
+                // I think its handy to flip in place for now, hence these commands
+                // 1., 2. prevent overlap and paste
+                // 3.     preserve location so you can flip repeatedly w/ hotkey
```

Comment:
Can you explain this a little better? I also think the contents of `flip_selection` could be put here directly.

Reply:
I put the contents of `flip_selection` there, but I'm not sure how I can explain it better :sweat_smile: 

When I first tried out flipping and pasting, the pixels that were underneath were still there, so you would get overlapped, which is why I added the Erase command.

Then I added Paste, so that the selection gets pasted and finally switch the Mode to make it easier to flip repeatedly

Reply:
I think if anything, the code is clear and I was confused by the comment :sweat_smile: